types/appointment.d.ts

```diff
@@ -0,0 +1,74 @@
+declare module 'skedify-sdk' {
+  export type AppointmentState =
+    | 'incoming'
+    | 'outgoing'
+    | 'completed'
+    | 'cancelled'
+    | 'accepted'
+    | 'occurred'
+
+  export interface Appointment extends Omit<BaseFields, 'enterprise_id'> {
+    state: AppointmentState;
+    contact_id: string;
+    customer_id: string;
+    subject_id: string;
+    office_id: string;
+    meeting_type: string;
+    first_possibility_id: string;
+    notes?: string;
+    outcome_remark?: string;
+    trailing_buffer_time: number;
+    metadata: any[];
+    customer_showed: boolean;
+    is_success: boolean;
+    video_url_for_contact: string;
+    video_url_for_customer: string;
+    video_test_url_for_contact: string;
+    video_test_url_for_customer: string;
+    cancellation_reason_id?: string;
+    cancelled_by_id?: string;
+    cancelled_by_type?: string;
+    initiated_from: string;
+    initiated_by_id: string;
+    initiated_by_type: string;
+    updated_at: string;
```

Comment:
Shouldn't `created_at` be added as well?

Reply:
Appointment extends from BaseFields, which types `id`, `created_at` and `external_id`